documentation/guides/getting-started.md

```diff
@@ -1,6 +1,10 @@
 # Getting started
 
-For the fastest possible start, check out the `@spectrum-web-components/bundle` package which includes all of the elements defined by Spectrum Web Components in one easy to import entry point. Copy and paste the following HTML sample, and you can be off and running.
+For the fastest possible start, check out the `@spectrum-web-components/bundle` package, which includes all of the elements defined by Spectrum Web Components in one easy-to-import entry point.
+
+## What you're getting
+
+The `@spectrum-web-components/bundle` package is literally _everything_ that Spectrum Web Components has to offer. This is why when bundled, <sp-link target="_blank" href="https://bundlephobia.com/result?p=@spectrum-web-components/bundle">it weighs as large as it does</sp-link>, and is why we DO NOT suggest leveraging this technique in a production application. That being said, it is the shortest path to getting right into developing with Spectrum Web Components, so dive right in. Copy and paste the following HTML sample, and you’ll be up and running. Good luck, and have fun!
```

Comment:
```suggestion
The `@spectrum-web-components/bundle` package is literally _everything_ that Spectrum Web Components has to offer. This is why when bundled <sp-link target="_blank" href="https://bundlephobia.com/result?p=@spectrum-web-components/bundle">it weighs as large as it does</sp-link> and is why we DO NOT suggest leveraging this technique in a production application. That being said, it is the shortest path to getting right into developing with Spectrum Web Components, so dive right in. Copy and paste the following HTML sample, and you’ll be up and running. Good luck, and have fun!
```
Was this too many commas, or not enough?

Reply:
I think a comma here is fine.